pom.xml

```diff
@@ -429,6 +369,27 @@
                 </configuration>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <version>2.8.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.maven.wagon</groupId>
+                        <artifactId>wagon-ssh-external</artifactId>
+                        <version>2.6</version>
```

Comment:
Why did you downgrade the version? [It was 2.8 before](https://github.com/osiam/resource-server/pull/53/files#diff-600376dffeb79835ede4a0b285078036L521).